src/main/flight/gps_rescue.c

```diff
@@ -315,10 +318,13 @@ static void rescueAttainPosition()
     // D component is error based, so includes positive boost when climbing and negative boost on descent
     float throttleDRaw = ((altitudeError - previousAltitudeError) / rescueState.sensor.altitudeDataIntervalSeconds);
```

Comment:
(maybe vertical speed can be used instead?)

Reply:
That will make it more readable.  Thanks.

Reply:
Or maybe `altitudeDerivative` ?